src/main/java/mods/railcraft/common/core/Railcraft.java

```diff
@@ -210,7 +213,7 @@ public void serverCleanUp(FMLServerStoppingEvent event) {
     public void missingMapping(FMLMissingMappingsEvent event) {
         for (FMLMissingMappingsEvent.MissingMapping mapping : event.get()) {
             if (mapping.type == GameRegistry.Type.BLOCK) {
-                Block block = GameRegistry.findBlock(MOD_ID, MiscTools.cleanTag(mapping.name));
+                Block block = GameRegistry.findBlock(MOD_ID, mapping.resourceLocation.getResourcePath());
```

Comment:
This bug is currently stopping BuildCraft from migrating 1.7.10 worlds and migrating fluids after I refactored the way that fluids were registered.

> On 30 Mar 2016, at 09:51, hea3ven <notifications@github.com> wrote:
> 
> In src/main/java/mods/railcraft/common/core/Railcraft.java:
> 
> > @@ -210,7 +213,7 @@ public void serverCleanUp(FMLServerStoppingEvent event) {
> >      public void missingMapping(FMLMissingMappingsEvent event) {
> >          for (FMLMissingMappingsEvent.MissingMapping mapping : event.get()) {
> >              if (mapping.type == GameRegistry.Type.BLOCK) {
> > -                Block block = GameRegistry.findBlock(MOD_ID, MiscTools.cleanTag(mapping.name));
> > +                Block block = GameRegistry.findBlock(MOD_ID, mapping.resourceLocation.getResourcePath());
> Just in case, to save you debugging headaches, this event is currently not working: MinecraftForge/MinecraftForge#2491
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
>